cmd/charm/charmcmd/grant.go

```diff
@@ -49,8 +49,7 @@ The --set parameters is used to overwrite any existing ACLs for the charm or bun
     charm grant ~johndoe/wordpress --acl write --set fred,bob
 
 When a specific charm or bundle revision is provided,
-the channel parameter is ignored. Otherwise the "stable"
-channel is used by default. To select another channel,
+the channel parameter is ignored. To select a channel,
```

Comment:
Something like this, perhaps:

    When a specific charm or bundle revision is provided
    and no channel parameter is explicitly given,
    the default channel for the charm or bundle will be used.